lib/oli_web/live/community_live/show_view.ex

```diff
@@ -219,28 +219,35 @@ defmodule OliWeb.CommunityLive.ShowView do
       is_admin: user_type == "admin"
     }
 
-    case Groups.create_community_account_from_email(user_type, email, attrs) do
-      {:ok, _community_account} ->
-        socket = put_flash(socket, :info, "Community #{user_type} successfully added.")
-        updated_assigns = community_accounts_assigns(user_type, attrs.community_id)
+    emails =
+      email
+      |> String.split(",")
+      |> Enum.map(&String.trim(&1))
 
-        {:noreply, assign(socket, updated_assigns)}
+    socket =
+      case Groups.create_community_accounts_from_emails(user_type, emails, attrs) do
+        {:ok, _community_accounts} ->
+          put_flash(socket, :info, "Community #{user_type}(s) successfully added.")
 
-      {:error, error} ->
-        message =
-          case error do
-            :author_not_found ->
-              "Community admin couldn't be added. Author does not exist."
+        {:error, error} ->
+          message =
+            case error do
+              :author_not_found ->
+                "Community admin couldn't be added. Author does not exist."
```

Comment:
We can have the case where more than one author fail, right? Maybe we can tweak a little bit the message to take this into account?

Reply:
FFI: discussed this via slack and decided to collapse all possible error types into one generic error message. This feature will be improved in a future story